Resources/Prototypes/_FarHorizons/Entities/Objects/Devices/flatpack.yml

```
@@ -22,4 +22,16 @@
     layers:
     - state: large
   - type: Flatpack
-    entity: Turbine
\ No newline at end of file
+    entity: Turbine
+
+- type: entity
+  parent: BaseFlatpack
+  id: PTKFlatpack
+  name: PTK-800 flatpack
```

Just beginning to enforce capitalization for names.
```suggestion
  name: PTK-800 Flatpack
```